### PR TITLE
Need bai for dexseq-count

### PR DESCRIPTION
**Pull request by TomHarrop**
I'm getting `[E::idx_find_and_load] Could not retrieve index file for '/mnt/user-data-volC/data13/c/e/b/dataset_cebddb26-4aa7-4051-9740-b79025ccddd4.dat'` running dexseq-count on Galaxy Australia.

I'm just pushing this as a quick fix for version 1.44 - the bioconductor package seems to be missing the htseq dependency for later versions (see https://github.com/galaxyproject/tools-iuc/pull/4295)

---

FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

**Review by bernt-matthias — COMMENTED** [review]
The version needs a bump.

**Comment by TomHarrop** [comment]
> The version needs a bump.

I'm working on that over here: https://github.com/bioconda/bioconda-recipes/pull/46916

File: tools/dexseq/dexseq_count.xml [hunk 1]
```diff
@@ -20,14 +20,16 @@
...
 #elif $mode.mode_select == "count":
+    ln -s -f '${mode.bamfile}' 'input.bam' &&
+    ln -s -f '${mode.bamfile.metadata.bam_index}' 'input.bam.bai' &&
     dexseq_count.py
         --format bam
         --paired $mode.paired
         --stranded $mode.stranded
         --minaqual $mode.qual
         --order $mode.order
         $mode.flattened_gtf_in
```
[the remark this answers is not in the document]
Comment (TomHarrop) [reply tools/dexseq/dexseq_count.xml]: thanks, added some

**Comment by TomHarrop** [comment]
@bernt-matthias I think the biocontainer is fixed now so I’ve bumped the version in the wrapper.

**Review by bernt-matthias — APPROVED** [review]

**Comment by bernt-matthias** [comment]
Thanks

**Pull request merged**